utils/build-icons/lib/config.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
         removeDimensions: true
       },
       {
-        removeAttrs: { attrs: '(stroke|fill)' }
+        removeAttrs: false
```

Comment:
I think we should still provide a whitelist here to establish convention  on how we actually target SVG nodes whether using CSS or JS.

It would also serve to remove clutter and potential selector mismatches though we should probably just be quite explicit with `className`.

Reply:
Not sure what you mean; what's wrong with using the `id` as target?

Reply:
My original comment was a little vague wrt "SVG nodes." I was referring to targeting nodes *within* an SVG, eg:

```svg
<svg>
  <symbol id="the-icon">
    <path d="[...]" />
    <path className="accent-color" d="[...]" />
    <path className="accent-color" d="[...]" />
  </symbol>
</svg>

<!-- [...] -->

<use class="prince" href="#the-icon"> 
```

```scss
// Default
#the-icon path.accent-color {
  stroke: $default-color;
  fill: $default-color;
}

// Prince's
#the-icon.prince path.accent-color {
  stroke: $prince-purple;
  fill: $prince-purple;
}
```

Reply:
I guess really just the same reason we don't use IDs outside of SVG?

Reply:
What's wrong with
```css
#the-icon.prince #thehat {....} 
```

The reason why I think this might be worth doing is because it's trivial for designers to name paths as they wish via `id`, and we don't have to do any work to add `className` manually editing SVGs.

I think it works?

https://codepen.io/anon/pen/ejVZjE - note in this example I didn't use `<use>` but it should work as expected. Right?